stumpy/core.py

```diff
@@ -2403,6 +2403,13 @@ def rolling_isconstant(a, w, a_subseq_isconstant=None):
 
     elif isinstance(a_subseq_isconstant, np.ndarray):
         isconstant_func = None
+        if a.ndim != a_subseq_isconstant.ndim:  # pragma: no cover
+            msg = (
+                "The arrays `a` and `a_subseq_isconstant` must have same "
```

Comment:
must have THE same

Reply:
I really need to start paying more attention to the article THE. Thanks a lot for pointing it out. I will fix it.